src/components/StaffList/index.jsx

```diff
@@ -73,6 +225,22 @@ class StaffList extends React.Component {
             )}
           </Droppable>
         </DragDropContext>
+        {/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */}
```

Comment:
couldn't figure out how to get this passed the linter

Reply:
Did you try moving the ```onKeyDown``` to the ```label``` or the ```Autosuggest``` ?

Reply:
Didn't try label, i'll try that next.  Autosuggest is third party component, how could I move it there?

Reply:
Sometimes third party components pass any extra props through, so it would be worth a shot.